src/test/resources/features/MFE/login-test.feature

```diff
@@ -0,0 +1,18 @@
+Feature: User Login check
+  I want to be able to see if a valid user login correctly
+
+ Background: User Opens visits login page
+    Given User opens the websites login page
+  Scenario Outline: User fill in credentials and login
+    
+    When User enters username "<username>"
+      And User clicks continue
+      And User enters Password "<password>"
+      And User Log in
+      And User selects unknown location
+      And User clicks confirm
+    Then User should arrive at home page
+
+    Examples:
+        | username | password | 
+        | admin  | Admin123  | 
```

Comment:
It's a good idea to always end a file with a blank line. This is because some POSIX tools make an assumption that files will have a blank final line.

Reply:
Thank you very much @ibacher for your suggestions. I find them very helpful